spring-session/src/main/java/org/springframework/session/data/cassandra/config/annotation/web/http/EnableCassandraHttpSession.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2014-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.session.data.cassandra.config.annotation.web.http;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import com.datastax.driver.core.ConsistencyLevel;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+/**
+ * Add this annotation to a {@code @Configuration} class to expose the
+ * SessionRepositoryFilter as a bean named "springSessionRepositoryFilter" and backed by
+ * Cassandra. The annotation properties can be used to control the way the cluster is used
+ * to store sessions.
+ *
+ * @author Andrew Fitzgerald
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+@Documented
+@Import(CassandraHttpSessionConfiguration.class)
+@Configuration
+public @interface EnableCassandraHttpSession {
+
+	/**
+	 * The name of cassandra table used by Spring Session to store sessions.
+	 * @return the cassandra table name
+	 */
+	String tableName() default "spring_session";
+
+	/**
+	 * The name of cassandra keyspace to connect to.
+	 * @return the cassandra keyspace
+	 */
+	String keyspace() default "";
```

Comment:
I'm not sure configuring the keyspace/contact points/port at this level is a good idea. Both are environmental settings that should be provided externally.

@vpavic @rwinch thoughts?